src/cascadia/TerminalSettingsModel/ActionArgs.idl

```diff
@@ -5,16 +5,14 @@ import "Command.idl";
 
 namespace Microsoft.Terminal.Settings.Model
 {
-    interface IActionArgs

```

Comment:
this seems unrelated, but I guess it's fine? haha

Reply:
Ah that's a mistake from invoking clang-format on it. I'll revert it.